implementations/micrometer-registry-dynatrace/src/test/java/io/micrometer/dynatrace/types/DynatraceTimerTest.java

```diff
@@ -50,24 +50,30 @@ class DynatraceTimerTest {
     private static final PauseDetector PAUSE_DETECTOR = new NoPauseDetector();
 
     @Test
-    void testHasValues() {
+    void testTimerCount() {
         DynatraceTimer timer = new DynatraceTimer(ID, CLOCK, DISTRIBUTION_STATISTIC_CONFIG, PAUSE_DETECTOR,
                 BASE_TIME_UNIT);
-        assertThat(timer.hasValues()).isFalse();
+
+        assertThat(timer.count()).isZero();
         timer.record(Duration.ofMillis(314));
-        assertThat(timer.hasValues()).isTrue();
+        assertThat(timer.count()).isEqualTo(1);
         timer.record(Duration.ofMillis(476));
-        assertThat(timer.hasValues()).isTrue();
+        assertThat(timer.count()).isEqualTo(2);
 
-        // checks that the recorded values are returned in the TimeUnit used to set up the
-        // instrument
-        assertMinMaxSumCount(timer, 314, 476, 790, 2);
-        assertThat(timer.hasValues()).isTrue();
-        assertMinMaxSumCount(timer.takeSummarySnapshotAndReset(), 314, 476, 790, 2);
-        assertThat(timer.hasValues()).isFalse();
+        timer.takeSummarySnapshotAndReset(BASE_TIME_UNIT);
+        assertThat(timer.count()).isZero();
 
```

Comment:
Any reason you split asserting the count + the recorded values? I was confused for a bit why we stopped asserting only to see the test below. 

Also empty line here.